CMakeLists.txt

```diff
@@ -23,7 +23,13 @@ endif()
 
 option(USE_SHARED_ENET "Use shared libenet if found rather than Dolphin's soon-to-compatibly-diverge version" OFF)
 option(USE_UPNP "Enables UPnP port mapping support" ON)
-option(ENABLE_NOGUI "Enable NoGUI frontend" ON)
+
+if (NOT WIN32)
```

Comment:
There's no real reason that Win32 can't have a nogui frontend. Just don't compile PlatformX11.cpp ;)

Reply:
Fact is that we currently don't provide any working platforms for Windows, so I don't think we should offer a non-functional option.